test/overseer/store/jdbc_test.clj

```diff
@@ -0,0 +1,156 @@
+
+(deftest test-job-dep-jdbc-maps
+  (let [j0 (test-utils/job {:job/id "j0"})
+        j1 (test-utils/job {:job/id "j1"})
+        j2 (test-utils/job {:job/id "j2"})
+
+        graph
+        (loom.graph/digraph
+          {j0 []
+           j1 [j0]
+           j2 [j1]})]
+    (is (= #{{:job_id "j1" :dep_id "j0"}
+             {:job_id "j2" :dep_id "j1"}}
+           (set (store/job-dep-jdbc-maps graph))))))
+
+(deftest test-transact-graph
+  (let [{:keys [db-spec] :as store} (test-store)
+        {job-id :job/id :as job-before} (test-utils/job)
+        _ (core/transact-graph store (api/simple-graph job-before))
+
+        job-after
+        (->> {:select [:*] :from [:overseer_jobs] :where [:= :id job-id]}
+             sql/format
+             (j/query db-spec)
+             first)]
+    (is (:created_at job-after) "It sets created_at when transacting new records")
+    (is (:updated_at job-after) "It sets updated_at when transacting new records")))
+
+(def ^:private sql-type-timestamp 93) ; From sql.h; corresponds to datetime
+
+(deftest test-install
+  (let [{:keys [db-spec] :as store} (test-store)]
+    (testing "tables"
+      (let [tables-by-name
+            (->> {:select [:*] :from [:information_schema.tables]}
+                 sql/format
+                 (j/query db-spec)
+                 (group-by (comp string/lower-case :table_name))
+                 (std/map-kv first))]
+        (is (contains? tables-by-name "overseer_jobs"))
+        (is (contains? tables-by-name "overseer_dependencies"))) )
+    (testing "columns"
+      (let [cols-by-name
+            (fn [table-name]
+              (->> {:select [:*] :from [:information_schema.columns] :where [:= :table_name table-name]}
+                   sql/format
+                   (j/query db-spec)
+                   (group-by (comp string/lower-case :column_name))
+                   (std/map-kv first)))
+
+            job-cols-by-name (cols-by-name "OVERSEER_JOBS")
+            dep-cols-by-name (cols-by-name "OVERSEER_DEPENDENCIES")]
+        (is (= "VARCHAR" (get-in job-cols-by-name ["id" :type_name])))
+        (is (= "VARCHAR" (get-in job-cols-by-name ["type" :type_name])))
+        (is (= "VARCHAR" (get-in job-cols-by-name ["args" :type_name])))
+        (is (= "TINYINT" (get-in job-cols-by-name ["status" :type_name])))
+        (is (= "VARCHAR" (get-in job-cols-by-name ["failure" :type_name])))
+        (is (= "INTEGER" (get-in job-cols-by-name ["heartbeat" :type_name])))
+        (is (= "INTEGER" (get-in job-cols-by-name ["lock_version" :type_name])))
+        (is (= sql-type-timestamp (get-in job-cols-by-name ["created_at" :data_type])))
+        (is (= sql-type-timestamp (get-in job-cols-by-name ["updated_at" :data_type])))
+
+        (is (= "VARCHAR" (get-in dep-cols-by-name ["job_id" :type_name])))
+        (is (= "VARCHAR" (get-in dep-cols-by-name ["dep_id" :type_name])))))))
+
+(deftest test-protocol
+  (store-test/test-protocol test-store))
```

Comment:
This looks like it's missing tests for transacting the graph, ensuring that the job got in with its dependencies etc.